crates/hdk/Cargo.toml

```diff
@@ -53,4 +53,7 @@ properties = ["holochain_zome_types/properties"]
 unstable-functions = [
   "holochain_zome_types/unstable-functions",
   "hdi/unstable-functions",
+  "unstable-countersigning",
```

Comment:
You don't need `unstable-functions` for countersigning any more. The only function `accept_preflight_request` has been moved to `unstable-countersigning`. I figured you'd want to include that unstable function when enabling `unstable-functions`, but it could both just be separate features too. No need to include in the other. I'll do that.